src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -217,6 +236,54 @@ public void processPresence(Presence packet)
             muc.addPresenceInterceptor(presenceInterceptor);
 
             muc.create(myNickName);
+            muc.addParticipantListener(new PresenceListener()
```

Comment:
Do you think it would make sense to move this listener to JitsiMeetConferenceImpl directly as nested class/anonymous or something ? It knows when the chat room is created and we would get rid of "jicofo" package import.